### ISSUE TITLE
Kicking People from groups

### ISSUE THREAD

**Issue by MarcusDunn** [CobeyH/SENG-480A#90]
* Feature name: Kicking People from groups
* Description: we want to be able to kick people from groups for being *toxic*
* Requires: Not much, remove them from the group and don't let them join again (or randomize the invite links)
* Status: Not done
* Time Estimation: an hour if we just ban them forever. Longer for more complex options.

**Comment by CobeyH** [comment]
I don't think this will improve the UX that much compared to some of our other issues. Maybe we should leave this for the final submission?
